rules/windows/process_creation/proc_creation_win_susp_conhost_option.yml

```diff
@@ -15,11 +15,12 @@ logsource:
     category: process_creation
 detection:
     selection:
+        IntegrityLevel: 'High'
```

Comment:
Need to upgrade
- title: Suspicious High IntegrityLevel Conhost Legacy Option
- modified: 2022/12/09
- description: add information that High is administrators thrust (best it to find a link to MS doc (not found)